src/security-council-mgmt/SecurityCouncilNominationsManager.sol

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "@openzeppelin/contracts-upgradeable/governance/IGovernorUpgradeable.sol";
+
+import "./SecurityCouncilManager.sol";
+import "./SecurityCouncilNominationsGovernor.sol";
+
+// todo: constructor or initializer
+contract SecurityCouncilNominationsManager {
+    // todo: set these in the constructor / initializer
+    uint256 public targetNomineeCount;
+    Cohort public firstCohort;
+    uint256 public firstNominationStartTime;
+    uint256 public nominationFrequency;
+    SecurityCouncilManager public securityCouncilManager;
+    SecurityCouncilNominationsGovernor public nominationsGovernor;
+
+    // number of nomination elections that have been created
+    uint256 public nominationsCount;
+
+    // maps electionId to mapping of candidates up for nomination
+    mapping(uint256 => mapping(address => bool)) public isCandidateUpForNomination;
+
+    // maps NominationsManager's electionId to NominationsGovernor's proposalId
+    mapping(uint256 => uint256) public electionIdToProposalId;
+
+    modifier onlyGovernor {
+        require(msg.sender == address(nominationsGovernor), "Only the governor can call this");
+        _;
+    }
+
+    function createElection() external returns (uint256 electionId) {
```

Comment:
I'm wondering if the manager and the elections governor can be merged into a single contract; i.e., have all of this logic in the governor's `propose`, have `nominateCandidate` (below) live on the governor (which is what I'd have expected). Might really just be an aesthetic question

Reply:
I think that'd make sense, would probably simplify some things like electionId <-> proposalId for example